docs/Calamari/Node Operators/get_calamari.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Can we change the name of this folder so the path doesn't have a space in it?

Reply:
Not sure how to make the section names independent from the file names.
@grenade

Reply:
> Not sure how to make the section names independent from the file names.
> @grenade

Nvm figured it out